core/mutator.ts

```diff
@@ -456,7 +454,7 @@ export class Mutator extends Icon {
       block.rendered = false;
 
       // Allow the source block to rebuild itself.
-      block.compose!(this.rootBlock_);
+      block.compose!(this.rootBlock);
```

Comment:
random aside: pretty sure we could have `block` say it is a type that describes a "Block where we know the compose property is present" since we know it actually must be present if we are calling this, using union types. that would be fancy and i'm only mentioning it here because i'm excited by how specific we can get with TS, not because i think we should actually change anything rn